netty/src/main/java/io/grpc/transport/netty/GrpcSslContexts.java

```diff
@@ -47,15 +50,30 @@
 public class GrpcSslContexts {
   private GrpcSslContexts() {}
 
-  private static ApplicationProtocolConfig DEFAULT_APN = new ApplicationProtocolConfig(
-          Protocol.ALPN,
-          SelectorFailureBehavior.FATAL_ALERT,
-          SelectedListenerFailureBehavior.FATAL_ALERT,
-          "h2",
-          "h2-17",
-          "h2-16",
-          "h2-15",
-          "h2-14");
+  private static String[] HTTP2_VERSIONS = {
+      "h2",
+      "h2-17",
+      "h2-16",
+      "h2-15",
+      "h2-14"};
+
+  public static ApplicationProtocolConfig ALPN = new ApplicationProtocolConfig(
```

Comment:
It doesn't seem like we should be exposing something like this, even to qps benchmark. If the benchmark is special in some way (which it doesn't seem to be), then we could just copy it. If the benchmark is not special, then we would be expecting users to use these.